### PR TITLE
553 (Turning c02 and O2) into environmental compounds.

### PR DESCRIPTION
**Pull request by Untrustedlife**

### CODE REVIEW

**Comment by Untrustedlife** [comment]
Okay final part of this is done, waiting on the reviews now.

File: scripts/microbe_stage/biome.as [hunk 1]
```diff
@@ -1,5 +1,5 @@
 // Global table which stores the current biome the player is in.
-uint64 currentBiome = 0;
+uint64 currentBiome = 42;
```
Comment (hhyyrylainen) [inline scripts/microbe_stage/biome.as]: This should be propably set to something like `1000000` to make sure that we won't have problems if we add a bunch of more biomes.

File: src/microbe_stage/generate_cell_stage_world.rb [hunk 2]
```diff
@@ -148,7 +148,9 @@
                      runtick: {group: 900, parameters: []}),
 
     EntitySystem.new("ProcessSystem", ["CompoundBagComponent", "ProcessorComponent"],
-                     runtick: {group: 10, parameters: []}),
+                     runtick: {group: 10, parameters: []},
+                     visibletoscripts: true,
+                     release: []),
```
Comment (hhyyrylainen) [inline src/microbe_stage/generate_cell_stage_world.rb]: Why is the release method added to the process system?

**Comment by hhyyrylainen** [comment]
I squashed a lot of the little commits to make this branch better to be merged (without the final squash).
So this is ready for merging once @Untrustedlife confirms nothing broke.

**Pull request merged**